hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/TestOzoneTokenIdentifier.java

```diff
@@ -282,9 +248,9 @@ public void testSymmetricTokenPerfHelper(String hmacAlgorithm, int keyLen) {
    * Test serialization/deserialization of OzoneTokenIdentifier.
    */
   @Test
-  public void testReadWriteInProtobuf() throws IOException {
+  public void testReadWriteInProtobuf(@TempDir Path baseDir) throws IOException {
     OzoneTokenIdentifier id = getIdentifierInst();
-    File idFile = new File(BASEDIR + "/tokenFile");
+    File idFile = baseDir.resolve("/tokenFile").toFile();
```

Comment:
I think test failure is caused by `/tokenFile` being resolved as absolute path, not a child of `baseDir`.

```suggestion
    File idFile = baseDir.resolve("tokenFile").toFile();
```

Reply:
Done. thanks.